scripts/navigation/Readme.md

```diff
@@ -0,0 +1,16 @@
+# About  
+Jest to kod napisany dla navigacji który pobiera wszystkie linki oraz elementy LIs (listy) i sprawdza żeby żaden link nie posiadal klasy "active", nastepnie gdy użytkownik kliknie na link w navigacji wtedy link zmieni kolor na (Hex #f7ac53).
```

Comment:
Damian - tutaj masz napisac po co to jest, a nie co on robi :)

Cos jak:
skrypt dodaje klase .active dla aktywnego linku w menu desktopowym oraz mobilnym.